README.md

```diff
@@ -18,32 +18,83 @@ The formatting and reannotation are based on information extracted from:
 - the 'filtered-results.csv' helper file as supplied to @sneumann during the [HUPO-PSI](http://www.psidev.info/hupo-psi-meeting-2020)
 
 
-Viewing the ISA-tab formatted PXD017710 with [ISATab-Viewer](https://sneumann.github.io/PXD017710/isaviewer-demo.html)
+Viewing the ISA-tab formatted and reannotated PXD017710 with [ISATab-Viewer](https://isa-tools.org/PXD017710/isaviewer-demo.html)
 
+Viewing the ISA-tab formatted and reannotated PXD017710 locally, do the following:
 
+```bash
+python -m http.server 8000
+```
+
+Then point your browser to `http://0.0.0.0:8000/isaviewer-demo.html`
 
 ### Task performed:
 
 * initial structure of the study design in ISA format:
 
 * linkage of Proteome and Translatome data (supplementary material) to ISA assay tables (via Derived Data File)
 
+* processing the Proteome and Translatome data (supplementary material) with python pandas library to generate the following csv files:
+
+	- proteome_intensities_long_table_ggplot2.txt
+	- proteome_diffanal_ratio_pvalue_long_table_ggplot2.txt
+	- translatome_intensities_long_table_ggplot2.txt	
+	- translatome_diffanal_ratio_pvalue_long_table_ggplot2
+	
+	The files are `long table` corresponding to a `melt` on the Excel file originally generated by the users and can be readily loaded in R ggplot2 library for graphical representation.
+	The statistical relevant elements have been annotated with the [STATO ontology](http://stato-ontology.org/) and the tables comply with a Frictionless.io Data Package.
+	The jupyter notebook for the transformation is available.
+
+* conversion of raw data to mzML format
+
+install docker: 
+```bash
+		>brew update
+		>brew install docker
+```
+
+sign in to docker
+```bash
+		>docker start
+		>docker login
+```
+
+pull docker container for ProteoWizard:
+```bash
+>docker pull chambm/pwiz-i-agree-to-the-vendor-licenses
+```
+
+:warning: be sure to sign-up and login to https://hub.docker.com/
+
+in order to be able to reach
+
+https://hub.docker.com/r/chambm/pwiz-skyline-i-agree-to-the-vendor-licenses
+
+
+run the pwiz tool from the container over the raw data:
+```bash
+ docker run -it --rm -e WINEDEBUG=-all -v /Users/philippe/Downloads/PXD017710/raw/:/data chambm/pwiz-skyline-i-agree-to-the-vendor-licenses wine msconvert /data/*.raw --mzML
```

Comment:
Did you mention that the download comes from 
https://www.ebi.ac.uk/pride/archive/projects/PXD017710
and/or ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2020/03/PXD017710/